x/pocketcore/types/proof.go

```diff
@@ -169,7 +171,8 @@ func (rp RelayProof) BytesWithSignature() []byte {
 		Token:              rp.Token.HashString(),
 	})
 	if err != nil {
-		panic(fmt.Sprintf("an error occured converting the relay RelayProof to bytes with signature:\n%v", err))
+		fmt.Println(fmt.Errorf("an error occured converting the relay RelayProof to bytesWithSignature:\n%v", err))
+		os.Exit(1)
```

Comment:
```suggestion
  log.Fatalf("an error occured converting the relay RelayProof to bytesWithSignature:\n%v", err)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)